spec/factories.rb

```diff
@@ -1,4 +1,7 @@
-FactoryGirl.define do
+FactoryGirl.define do  factory :admin do
+    
+  end
```

Comment:
`end` at 3, 2 is not aligned with `factory :admin do` at 1, 23 or `FactoryGirl.define do  factory :admin do` at 1, 0